.buildkite/pipeline.yml

```
@@ -9,22 +9,33 @@ env:
 
 steps:
   - label: ":face_with_peeking_eye: Lint"
-    command: "make lint"
+    command: ".buildkite/run_linter.sh"
+    env:
+      PYTHON_VERSION: "3.10" # We don't really care about linter on multiple versions for now
```

```suggestion
      PYTHON_VERSION: "3.11" # We don't really care about linter on multiple versions for now
```
I'd vote we actually move to whatever the latest is that we support.